src/resources/items/ItemsReport.tsx

```diff
@@ -165,6 +165,18 @@ const referenceFields: RefFieldType[] = [
     resource: constants.R_BATCHES,
     source: 'batchNumber',
     label: 'Batch'
+  },
+  {
+    name: 'project',
+    resource: constants.R_PROJECTS,
+    source: 'name',
```

Comment:
the label for `Project` should come from the config data, like here:
`ConfigData?.projectName`